### ISSUE TITLE
'Micro' template petition

### ISSUE THREAD

**Issue by baselbers** [baselbers/woocommerce-pdf-invoices#162]
In version 2.7.0 a new template called 'Minimal' has been released and because of a complete code refactoring of the plugin, the 'Micro' template will no longer be supported.

Still preferring the 'Micro' template? Leave a 👍 to support the petition. We will consider to renew the 'Micro' template when there are enough votes.

**Comment by izirezo** [comment]
Yes, I wishes you to keep the "Micro" template. I do not like at all "Minimal"!!!

**Comment by krit86** [comment]
Of course, Yes!, I'm using Micro template for years and it looks more attractive compared to Minimal.
IMO, the Micro is the featured template of this plug-ing.

**Comment by Frank-Poupart** [comment]
Please keep supporting Micro.

**Comment by baselbers** [comment]
+1

**Comment by Arkymedes** [comment]
Whoa what? The most beautiful template for an Invoice will have the support dropped???

Get my vote!
